tasks/kibana.yml

```diff
@@ -26,23 +34,6 @@
     - reload systemd
     - restart kibana
 
-- name: Extract Kibana tarball.
-  unarchive:
-    copy: no
-    src: /opt/kibana-{{ elk_kibana_version }}-linux-x64.tar.gz
-    dest: /opt/
-    owner: "{{ elk_nginx_user }}"
-    group: "{{ elk_nginx_user }}"
-  when: kibana_directory_check.stat.exists == false
-
-- name: Symlink Kibana directory.
-  file:
-    state: link
-    dest: /opt/kibana
-    src: /opt/kibana-{{ elk_kibana_version }}-linux-x64
-    owner: "{{ elk_nginx_user }}"
-    group: "{{ elk_nginx_user }}"
-
   # Two-pass approach to creating logfile idempotently. Create the empty logfile
   # if none exists (force: no), but to ensure permissions we need a second task.
 - name: Create logfile for Kibana.
```

Comment:
Nope, there's nothing related to logging in the .deb package hooks. This is in the "start" function of the init script:
```
  # Ensure the log directory is setup correctly.
  [ ! -d "/var/log/kibana/" ] && mkdir "/var/log/kibana/"
  chown "$user":"$group" "/var/log/kibana/"
  chmod 755 "/var/log/kibana/"
```